minlz_test.go

```
@@ -33,8 +33,10 @@ import (
 	"github.com/klauspost/compress/zstd"
 )
 
-const maxUint = ^uint(0)
-const maxInt = int(maxUint >> 1)
+const (
+	maxUint = ^uint(0)
```

golangci-lint (2.12.2)

[error] const maxUint is unused

(unused)